.github/helpers/contracts/dummy-dao/createDao.ts

```diff
@@ -44,30 +45,64 @@ async function createDao() {
 
   console.log('ipfs cid', cid.path);
 
-  const metadata = ethers.utils.hexlify(ethers.utils.toUtf8Bytes(cid.path));
-  const daoConfig = [daoName, metadata];
-  const votingSettings = ['200000000000000000', '400000000000000000', '60000'];
+  let metadata = ethers.utils.hexlify(ethers.utils.toUtf8Bytes(cid.path));
+  let daoConfig = [daoName, metadata];
+  let votingSettings = ['200000000000000000', '400000000000000000', '60000'];
+
+  let overrides = {};
+
+  if (networkName === 'mumbai') {
```

Comment:
move this into its own file and replace all occurrences with an import and function call to this new file.
Also, add this gas estimation for polygon mainnet too and integrate it into the `hardhat.config.ts` so we can remove the hardcoded gas fee in `networks.json`

Reply:
- moved it out.
- i couldn't find a way to async update hardhat config with gas data, 
do u think it is possible?